pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -126,6 +127,26 @@ func (m *podContainerManagerImpl) GetPodContainerName(pod *v1.Pod) (CgroupName,
 	return cgroupName, cgroupfsName
 }
 
+func (m *podContainerManagerImpl) GetPodCgroupMemoryLimit(pod *v1.Pod) (uint64, error) {
+	podCgroupName, _ := m.GetPodContainerName(pod)
+	return m.cgroupManager.GetCgroupMemoryLimit(podCgroupName)
+}
+
+func (m *podContainerManagerImpl) GetPodCgroupCpuLimit(pod *v1.Pod) (int64, uint64, uint64, error) {
+	podCgroupName, _ := m.GetPodContainerName(pod)
```

Comment:
error checking?

Reply:
It does not return an error - the consuming function does return error if there is a parameter problem. Here's the precedent from upstream master 1.18 codebase: 

`root@fw0000358:~/MASTER_K8S# git grep -A3 GetPodContainerName
pkg/kubelet/cm/pod_container_manager_linux.go:  podContainerName, _ := m.GetPodContainerName(pod)
pkg/kubelet/cm/pod_container_manager_linux.go-  return m.cgroupManager.Exists(podContainerName)
pkg/kubelet/cm/pod_container_manager_linux.go-}
pkg/kubelet/cm/pod_container_manager_linux.go-

pkg/kubelet/cm/pod_container_manager_linux.go:  podContainerName, _ := m.GetPodContainerName(pod)
pkg/kubelet/cm/pod_container_manager_linux.go-  // check if container already exist
pkg/kubelet/cm/pod_container_manager_linux.go-  alreadyExists := m.Exists(pod)
pkg/kubelet/cm/pod_container_manager_linux.go-  if !alreadyExists {
`